test/config/config.js

```diff
@@ -29,7 +29,6 @@ module.exports = {
     host:     process.env.SEQ_MSSQL_HOST || process.env.SEQ_HOST || 'mssql.sequelizejs.com',
     port:     process.env.SEQ_MSSQL_PORT || process.env.SEQ_PORT || 1433,
     dialectOptions: {
-      instanceName:  process.env.MSSQL_INSTANCE ? process.env.MSSQL_INSTANCE : 'SQLEXPRESS',
```

Comment:
Okay, was just curious. I'm not familiar with MSSQL, I just ported the thing over to AppVeyor and copied a lot from tedious PowerShell setup script.